code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -550,18 +550,31 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
 	documentation = "Holds camera static images. Usually only visible to people who can well, see static.\
 		<br>We use images rather then vis contents because they're lighter on maptick, and maptick sucks butt."
 	plane = CAMERA_STATIC_PLANE
-	start_hidden = TRUE
 
 /atom/movable/screen/plane_master/camera_static/show_to(mob/mymob)
-	// If we aren't an AI, we have no need for this plane master (most of the time, ai eyes are weird and annoying)
-	if(force_hidden && isAI(mymob))
-		unhide_plane(mymob)
 	. = ..()
 	if(!.)
 		return
-	if(isAI(mymob))
+	var/datum/hud/our_hud = home.our_hud
+	if(!our_hud)
```

Comment:
could we use isnull here instead for the swag explicit readability

Reply:
fine, for you anything